README.md

```diff
@@ -1,41 +1,43 @@
-# A starter template for Run On Slack apps
+# Deno Starter Template for Run On Slack apps
 
-This repo is a starter template for new Run On Slack apps. 
+This repo is a starter template for new Run On Slack apps. This template uses Deno and the Slack CLI.
 
-## Prerequisites
+Before getting started, make sure you have a development workspace where you have permissions to install apps. If you don’t have one set up, go ahead and [create one](https://slack.com/create).
+
+## Installation
+
+#### Prerequisites
 
 To use this template, you will need to have installed and configured the Slack CLI. 
 Do this by following our [Quickstart Guide](https://api.slack.com/future/quickstart).
 
-# Making an app
+#### Get Started
 
-Make a Run On Slack app with this with repo by **creating an app from this template**, 
-**configuring your app**, then **writing functions**. 
+Make a Run On Slack app with this with repo by **creating an app from this template**, **configuring your app**, then **writing functions**. 
 
-Once you're done, you can Run your app with the CLI's local development server
-or Deploy your app to production.
+Once you're done, you can run your app with the CLI's local development server or Deploy your app to production.
 
-## 1. Creating an app from this template
-
-Clone this repo with the Slack CLI:
+### Setup Your Project
 
 ```zsh
-$ slack create my-app -t slack-samples/deno-starter-template
-```
+# Clone this project onto your machine
+slack create my-app -t slack-samples/deno-starter-template
 
-## 2. Configuring your app
+# Change into this project directory
+cd my-app
 
-The first thing we'll do is configure our app's manifest. This will let us 
-configure things like our app's name and the [scopes](https://api.slack.com/scopes)
-it requires.
+# Run app locally
```

Comment:
@misscoded - would love to get your thoughts on including the run + deploy instructions under this heading as well when there's also more in-depth information about how to run and deploy later on in this doc! having at least the run instructions here would match similar templates like bolt JS but is it redundant when we have a dedicated section related to running the app?

Reply:
I don't think it's redundant, but I could be wrong. Eager developers don't read documentation very well (I am almost always guilty of this), so having a section that very clearly indicates how it works might be useful.